lib/clients/utils.js

```diff
@@ -1,6 +1,8 @@
 const S3 = require('aws-sdk/clients/s3');
 const BackbeatClient = require('./BackbeatClient');
 
+const TWO_MINUTES = 120000;
```

Comment:
can we rename this to something appropriate? and may be the value be represented like `1000 * 60 * 2` if we want to know it is two minutes?